## E0106 (return-arg-in-generator)

### :x: Problematic code:

```python
def foo():
  i = 5
  a = 10
  yield i
  return a
```

### :heavy_check_mark: Correct code:

```python
def foo():
  i = 5
  yield i
```

### Rationale:

Occurs when a `return` statement is also used in addition to a `yield`
statement(s) in a function or method.

### Related resources:

- [Issue Tracker](https://github.com/PyCQA/pylint/issues?q=is%3Aissue+%22return-arg-in-generator%22+OR+%22E0106%22)
